ClickHaskell-tables/lib/ClickHaskell/Tables.hs

```diff
@@ -161,132 +164,34 @@ renderTableParameters :: [Builder] -> Builder
 renderTableParameters (parameter:ps) = "(" <> foldr (\p1 p2 -> p1 <> ", " <> p2) parameter ps <> ")"
 renderTableParameters []             = ""
 
+data PList (ps :: [Type]) where
```

Comment:
The name PList is a poor choice because the `P` prefix is ​​unimpressive and the list doesn't convey important semantics